recipes/pypif/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pypif" %}
+{% set version = "2.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f4bea4989a66f776b046180889a71f19f84e4b3fc93102d3a3f2f859d9866531
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - pypif
+  requires:
```

Comment:
If these are required to import `pypif` should they be runtime dependencies instead?

Reply:
Agreed